src/fill_wikipedia.py

```diff
@@ -0,0 +1,198 @@
+import click
+import requests
+import osmapi
+import overpy
+import getpass
+import re
+from tqdm import tqdm
+from colorama import Fore, Style
+
+
+def get_links(ident)->dict:
+    response = requests.get(f"https://www.wikidata.org/wiki/Special:EntityData/{ident}.json")
+    data = response.json()
+    ret = {}
+    wikidata_id = list(data["entities"].keys())[0]
+
+    for sitelink in data["entities"][wikidata_id]["sitelinks"].keys():
+        if sitelink.endswith("wiki") and sitelink!= "commonswiki":
+            ret[sitelink.replace("wiki","")] = data["entities"][wikidata_id]["sitelinks"][sitelink]["title"]
+
+    return {
+        "id": wikidata_id,
+        "langs": ret }
+    
+
+def filter_wikidata_tag(data)->dict:
+    ret = {
+        "nodes": [],
+        "ways": [],
+        "relations": []
+    }
+
+    for node in data.nodes:
+         if "wikidata" in node.tags:
+             ret["nodes"].append(node)
+
+    for way in data.ways:
+         if "wikidata" in way.tags:
+             ret["ways"].append(way)
+
+    for relation in data.relations:
+        if "wikidata" in relation.tags:
+            ret["relations"].append(relation)
+
+    return ret
+
+@click.command()
+@click.option("--debug",default=False,is_flag=True)
+def fill_wikipediacommand(debug):
+    
+    overpass_api = overpy.Overpass()
+
+    user = input("User:")
+    password = getpass.getpass("Password:")
+
+    area = input("Bounding box(South,West,North,East) or name value:")
+    default_lang = input("Default lang(ca,es,en):")
+    all_langs = bool(input("Add all available languages?(Y/n):") in ["y","Y","","yes"])
```

Comment:
Default to a non-recommended practice (https://wiki.openstreetmap.org/wiki/Key:wikipedia#Secondary_languages). In almost all cases, a single wikipedia tag is sufficient. Default to No